.github/issue_templates/Bug_reporting.md

```diff
@@ -0,0 +1,25 @@
+    Where did it occur?
```

Comment:
```suggestion
## Description of the issue
```
I'm not sure what you mean by "where" here. Perhaps start this out with letting the user just put in a brief description and then follow with your more precise requests below.